### PR TITLE
Attempt Windows build

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/install-libxgboost.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+. activate "${BUILD_PREFIX}"
```
Comment (beckermr) [inline recipe/install-libxgboost.sh]: Do you know what is going on here? This was in the upstream recipe too IIRC, but IDK why.

Comment (xhochy) [reply recipe/install-libxgboost.sh]: Seems like some paths are missing during copy but weirdly the CI build is behaving differently to my local installation, so I first need to find out what's the difference.

**Comment by asad1947** [comment]
Hi, We would like to have a windows version of xgboost. It seems like because of this open issue we cannot have xgboost in windows. Please let us know when this issue will get resolved? Thanks.

**Comment by xhochy** [comment]
> Hi, We would like to have a windows version of xgboost. It seems like because of this open issue we cannot have xgboost in windows. Please let us know when this issue will get resolved? Thanks.

Pull requests are welcome!

File: recipe/install-libxgboost.sh (hunk 1 above)
Comment (asad1947) [reply recipe/install-libxgboost.sh:3]: any update?

**Comment by wagnerpeer** [comment]
Hi @xhochy
would you be so nice and update your branch and retry the merge. I would like to have a look in the log of the failing windows build. However, the build is that old, that the logs are no longer available.
Is there an alternative way for me to re-run the build pipeline, except of forking and doing a PR myself? I would like to prevent a duplicate PR containing your work.

**Comment by jamesmyatt** [comment]
@conda-forge-admin, please rerender

**Comment by jamesmyatt** [comment]
@wagnerpeer, if I were you, I'd copy this to a new branch in my own fork and rebase it on the current master.

**Comment by PrimozGodec** [comment]
I moved changes from there to https://github.com/conda-forge/xgboost-feedstock/pull/59 just to see what are errors

**Pull request closed without merging**